Index: configure
===================================================================
--- configure	(revision 38364)
+++ configure	(working copy)
@@ -9009,6 +9009,7 @@
 CONFIG_SNAPPY = yes
 CONFIG_STARTCODE = yes
 CONFIG_TEXTUREDSP = yes
+CONFIG_TEXTUREDSPENC = yes
 CONFIG_VC1DSP = yes
 CONFIG_VIDEODSP = yes
 CONFIG_VP3DSP  = yes
@@ -9686,6 +9687,7 @@
 #define CONFIG_SNAPPY 1
 #define CONFIG_STARTCODE 1
 #define CONFIG_TEXTUREDSP 1
+#define CONFIG_TEXTUREDSPENC 1
 #define CONFIG_VC1DSP 1
 #define CONFIG_VIDEODSP 1
 #define CONFIG_VP56DSP 1
